share/lib/python/scripts/_binwrapper.py

```diff
@@ -43,4 +46,19 @@ def _config_exe(exe_name):
 
 if __name__ == '__main__':
     exe = _config_exe(os.path.basename(sys.argv[0]))
-    os.execv(exe, sys.argv)
+
+    # special is a now a full-on binary. therefore, we've wrapped around special
+    # in order to still be able to configure NEURON environment variables (run above)
+    if exe.endswith('special'):
+        # point executable to `special.nrn`
+        exe = os.path.join(sys.argv[0]+'.nrn')
+
+    subprocess.call([exe, *sys.argv[1:]])
+
+    # wrap around special for nrnivmodl, since special is now a full-on binary
+    #   special >> special.nrn
+    #   nrniv > special
+    if exe.endswith('nrnivmodl'):
+        print('Wrapping special binary')
+        shutil.move(os.path.join(platform.machine(), 'special'), os.path.join(platform.machine(), 'special.nrn'))
```

Comment:
I will let @ferdonline / @nrnhines to comment about `platform.machine()` : Is `CMAKE_SYSTEM_PROCESSOR` from CMake would be same as `platform.machine()` ?

Reply:
Not sure of what the alternative would be for this

Reply:
**not** suggesting as a solution **but** the directory under `os.environ["NEURONHOME"]/demo/release/` is equivalent to `CMAKE_SYSTEM_PROCESSOR`:
```
→ ls -lrt ./lib/python3.9/site-packages/neuron/.data/share/nrn/demo/release/
total 104
...
drwxr-xr-x  29 kumbhar  10067   928 May 25 12:44 x86_64
```

Reply:
As mentioned, since this is not guaranteed to be the same, I would maybe append a list of common directories and look for `special` inside?